scripts/fonts/add-polish-glyphs.py

```
@@ -0,0 +1,81 @@
+
+# smaller dot for ż/Ż, centered over z, placed just above the letter
+make(0x017C,"z","dot.sm",337.5-DOTSM_C,678,675)    # ż  dot at 678-790 (a bit more gap above x-height)
+make(0x017B,"Z","dot.sm",337.5-DOTSM_C,802,675)    # Ż  dot at 802-914 (a bit more gap above cap)
+
+# ogonek (bottom-right), targets to verify visually
+make(0x0105,"a","ogonek",440-OGO_C,0,667)  # ą
+make(0x0119,"e","ogonek",420-OGO_C,0,665)  # ę
+make(0x0104,"A","ogonek",450-OGO_C,0,667)  # Ą
+make(0x0118,"E","ogonek",380-OGO_C,0,583)  # Ę
+
+f.selection.all(); f.unlinkReferences(); f.round()
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)